README.md

```diff
@@ -18,15 +18,15 @@ npm install prettysize
 
 ```javascript
 
-var pretty = require('prettysize');
+const pretty = require('prettysize');
```

Comment:
Not sure that I like all the `const` in the README. This is actually an error since you can't redefine `str` to a new value. This may make better sense as `let` for all of them. Or we change the `const str` to `const str1`, `const str2`, `const str3`, etc..

Reply:
Yes your right. tell me if you prefer one let, and reassignemenet, or different const var

Reply:
Let's do a single `let` and then just do the simple reassignment.

Reply:
done